### DISCUSSION TITLE
[FR][4.x]: EVENT_DEFINE_EDITOR_CONTENT in User and GlobalSet edit pages

### DISCUSSION THREAD

**Discussion by wbrowar** [craftcms/cms#11034] [Ideas] [answered]
@brandonkelly Thanks, for adding [the `ElementsController::EVENT_DEFINE_EDITOR_CONTENT` event](https://github.com/craftcms/cms/commit/b7b97c5fbcddc48d75c7a22af1011e37e06066c1) (and for even using Guide as a way to illustrate it). It works great on Entries, Assets, and Categories edit pages.

Could we add it to GlobalSet and User edit pages, as well (or a similar event with the same purpose)?

**Comment by brandonkelly** [comment, marked as answer]
Other element types will gain that event as things move over to using the unified element editor. Currently that’s not really possible for global sets or users, so those pages continue to support template hooks instead.

**Reply by wbrowar** [reply]
Oh, I thought those had all been removed. I just added the code back in and the hooks work for now. Thanks!

**Reply by brandonkelly** [reply]
Yep! And to be clear, we won’t break those until at least 5.0.
